tests/test_synthetic_pii.py

```diff
@@ -478,9 +478,12 @@ def test_ssn_mapping():
     simulants["has_ssn"] = [True if n % 2 == 0 else False for n in range(num_unique_ids)]
 
     # Get the map
-    ssn_map = get_simulant_id_maps(
-        "simulant_id", {"silly_observer": simulants}, None, randomness
-    )["ssn"]
+    try:
```

Comment:
Nice I like this.

Reply:
Seemed like a good compromise. If we want this functionality everywhere then we can add to vct or something.